libs/lib_presentation/src/main/kotlin/com/bael/dads/lib/presentation/di/ActivityNameQualifier.kt

```diff
@@ -0,0 +1,12 @@
+package com.bael.dads.lib.presentation.di
+
+import javax.inject.Qualifier
+
+@Qualifier
+@Retention(AnnotationRetention.BINARY)
+annotation class ActivityNameQualifier(val name: String) {
+
+    companion object {
+        const val ACTIVITY_MAIN = "main"
```

Comment:
Could you please explicitly state the data type? I know this sounds ridic, but it's just part of my convention :)